From: Erik Arfvidson <erik.arfvidson@unisys.com>
Date: Tue, 17 Nov 2015 13:34:50 -0500
Subject: staging: unisys: iochannel fix spacing around operators
Patch-mainline: v4.5-rc1
Git-commit: c06a278344ad5024e44d962ce9aa2ed945f0d1cf
References: bsc#1033962

This patch fixes check warning from checkpatch.pl in the macro definition
CHECK: spaces preferred around that '+' (ctx:VxV)

Signed-off-by: Erik Arfvidson <erik.arfvidson@unisys.com>
Signed-off-by: Benjamin Romer <benjamin.romer@unisys.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 drivers/staging/unisys/include/iochannel.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/unisys/include/iochannel.h
+++ b/drivers/staging/unisys/include/iochannel.h
@@ -380,7 +380,8 @@ struct net_pkt_xmtdone {
  */
 #define RCVPOST_BUF_SIZE 4032
 #define MAX_NET_RCV_CHAIN \
-	((ETH_MAX_MTU+ETH_HEADER_SIZE + RCVPOST_BUF_SIZE-1) / RCVPOST_BUF_SIZE)
+	((ETH_MAX_MTU + ETH_HEADER_SIZE + RCVPOST_BUF_SIZE - 1) \
+	/ RCVPOST_BUF_SIZE)
 
 struct net_pkt_rcvpost {
 	    /* rcv buf size must be large enough to include ethernet data len +
